pages/clubs/[id].vue

```diff
@@ -0,0 +1,64 @@
+<script setup lang="ts">
+import { useRoute } from 'vue-router'
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from '~/components/ui/card'
+import json from '~/content/clubs.json'
+import type { Clubs } from '~/content/clubs'
+
+const clubs: Clubs = json as Clubs
+const route = useRoute()
+const id = route.params.id // Fetch current Club ID via route params
+
+// Filter clubs based on C_GroupsID
+// It just works ;)
+const filteredClubs = Object.values(clubs).flatMap(clubCategory =>
+  clubCategory.flatMap(club => club.groups.filter(group => group.C_GroupsID === id)))
+
+// Get the number of members in each group
+// const groupMemberCounts = filteredClubs.map(group => group.gmember.length)
+
+// This page requires login
+definePageMeta({
+  middleware: ['auth'],
+})
+</script>
+
+<template>
+  <div v-if="filteredClubs.length > 0">
+    <div v-for="group in filteredClubs" :key="group.C_GroupsID">
+      <div class="flex">
+        <Card class="w-3/4">
+          <CardHeader>
+            <CardTitle class="flex items-center gap-x-1">
+              {{ group.C_NameC }}
+            </CardTitle>
+            <CardDescription>{{ group.C_NameE }}</CardDescription>
+          </CardHeader>
+          <CardContent>
+            <p>{{ group.C_DescriptionC }}</p>
+            <br>
+            <p>{{ group.C_DescriptionE }}</p>
```

Comment:
1. Only show `C_DescriptionC`. Add `C_DescriptionE` until we have added support for i18n.

2. Show "暂无简介" if `C_DescriptionC` is empty. 

3. Some clubs like the student union add HTML codes to the club description. We should clean the up by regex. Would you prefer cleaning it in the python script (of crawling the clubs) or cleaning it in the frontend?

Reply:
My answer to the 3rd question:

I would prefer cleaning it in the Python script. Since certain JavaScript scripts / CSS styles will only be available on TSIMS, processing the text on the frontend will not only increase the workload of clients' browser, but it will also create an amount of unused network traffic.

Reply:
Well, I've implemented such features with cleanHTML() function, so it can be fully accomplished via frontend rn